.github/workflows/abacus-playwright-tests.yml

```
@@ -44,6 +44,12 @@ jobs:
           kubernetes-version: 1.24.1
       - name: Run tilt
         run: |-
+          export BACKEND_LATEST_VERSION=$(skopeo list-tags docker://ghcr.io/opentdf/charts/backend \
+          | python3 -c "import sys, json; sys.stdout.write(json.load(sys.stdin)['Tags'][-1])")
```

Is `jq` included in the GitHub action runner?
```suggestion
          | jq -r '.Tags[-1]'
```

---

Nope))). I was going for it at first too))